aptos-move/framework/aptos-framework/sources/TypeInfo.move

```diff
@@ -21,9 +21,9 @@ module AptosFramework::TypeInfo {
 
     #[test]
     fun test() {
-        let type_info = type_of<AptosFramework::Account::Account>();
+        let type_info = type_of<TypeInfo>();
```

Comment:
why this change?

Reply:
This is because of cyclic dependency Account -> TestCoin -> Coin -> TypeInfo -> Account. It's not worth it for a test :)

Reply:
this is unintuitive to me.

Reply:
It doesn't make a lot of sense to introduce a dependency just for a unit test. If we want to keep Account, we should separate this test out into a different file.

Reply:
I imagine this dependency isn't even being compiled in. Lots of tests pull in dependencies just for testing.

Reply:
Compiler complained about cyclic dependency before, which was the reason why I had to change this.